python/py-cattrs/Portfile

```diff
@@ -0,0 +1,31 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-cattrs
+version             1.8.0
+revision            0
+platforms           darwin
+license             MIT
+maintainers         nomaintainer
+
+description         cattrs is an open source Python library for structuring and unstructuring data.
+long_description    cattrs is an open source Python library for structuring and unstructuring data. \
+                    cattrs works best with attrs classes, dataclasses and the usual Python collections, \
+                    but other kinds of classes are supported by manually registering converters.
+
+homepage            https://github.com/python-attrs/cattrs
+
+python.versions     37 38 39 310
+
+checksums           rmd160  7cc352191d51c4cfbedc1eacf8ad5b61045736ca \
+                    sha256  5c121ab06a7cac494813c228721a7feb5a6423b17316eeaebf13f5a03e5b0d53 \
+                    size    22501
+
+if {${name} ne ${subport}} {
+    depends_build-append \
```

Comment:
this should likely be `depends_lib-append` or is the dependency not needed at runtime anymore (I suspect it is)

Reply:
but you do need a build dependency on `py-setuptools` as it is imported in the `setup.py`